src/modules/universe/RadixBootstrapConfig.ts

```diff
@@ -0,0 +1,7 @@
+import { RadixUniverseConfig, RadixNodeDiscovery } from '../..';
+
+export interface RadixBootstrapConfig {
+    universeConfig: RadixUniverseConfig,
+    nodeDiscovery: RadixNodeDiscovery,
+    finalityTime: number,
```

Comment:
What's this `finalityTime`? It seems nice if it is what I think it is.

Reply:
Finality time is the delay, from receiving the last atom status update, after which the atom status can be considered final - it kind of depends on the network size, but 5s is a reasonable assumption for now.